tools/commands.js

```diff
@@ -486,7 +486,21 @@ main.registerCommand({
     utils.validatePackageNameOrExit(
       packageName, {detailedColonExplanation: true});
 
-    var fsName = colonConverter.convert(packageName);
+    // When we create a package, avoid introducing a colon into the file system
+    // by naming the directory after the package name without the prefix.
+    var fsName = packageName;
+    if (packageName.indexOf(":") !== -1) {
+      var split = packageName.split(":");
+
+      if (split.length > 2) {
+        Console.error(packageName +
```

Comment:
I thought:

> so I think that should be fine

Meant there was nothing to be done.

If you mean stop packages from ever having multiple colons in the `name` field, I'm not sure that's a good idea? I know for a fact there are some people using that as their package name, and it would be weird if this release simply stopped compiling their packages because of new name restrictions.